### ISSUE TITLE
[Tune] what the reward_attrs can be choose? (question)

### ISSUE THREAD

**Issue by chjq201410695** [ray-project/ray#3388]
I just see reward_attrs='mean_accuracy' and 'episode_reward_mean' in examples. I wonder if there are other options for regression task. and, if I use 'loss' here, is it means the loss of compiled keras model?

**Comment by richardliaw** [comment]
Hi; we recently updated the docs to include these. There are some magic fields where the console will provide output, but you are also able to add any arbitrary custom metric:

https://ray.readthedocs.io/en/latest/tune-usage.html#auto-filled-results

Let me know if you have any other questions.

**Comment by chjq201410695** [comment]
Thanks for your response. that's very helpful!

**Comment by chjq201410695** [comment]
> Hi; we recently updated the docs to include these. There are some magic fields where the console will provide output, but you are also able to add any arbitrary custom metric:
> 
> https://ray.readthedocs.io/en/latest/tune-usage.html#auto-filled-results
> 
> Let me know if you have any other questions.
sorry , there is a more question.
what's the meanning of 'mean_loss'. if its mse? or rmse? or the average of loss that compiled keras model provided?

**Comment by richardliaw** [comment]
You can provide it yourself (and define it to be whatever you want).

**Comment by chjq201410695** [comment]
> You can provide it yourself (and define it to be whatever you want).

OK . but what's the default definition? or 'mean_loss' is just a name? I must define it by myself. if I don't provide a loss function, and I fill 'mean_loss' to reward_attr. what will happen.

**Comment by richardliaw** [comment]
The default is (ideally) some decreasing function. If you fill `mean_loss` to be your MSE loss and put `reward_attr="mean_loss"` like in the examples, then some schedulers will not work because `reward_attr` is expected to be monotonically _increasing_. 

Let me know if you have any more questions.

**Comment by chjq201410695** [comment]
Ok, the reward_attr need a monotonically increasing loss function to minimize. I have seen it in document :)

The confusion for me is that "mean_loss", "mean_accuracy" and "neg_mean_loss" are just names or they have real meaning.

as you said, I guess the "mean_loss" is just a name, and  I should replace the "mean_loss" with a real loss function, right?     but in the examples, they just write "mean_accuracy" to reward_attr. and these examples work.
       
For example, I build a model in keras. I set :

        model.compile(loss=keras.losses.mean_squared_error, optimizer='sgd'). 

now, I use ray.tune to fine-tune the model.  how can I set the reward_attr in  the Scheduler/Search Algorithm? 
   ①  reward_attr = -keras.losses.mean_squared_error; 
   ②  reward_attr = "neg_mean_loss"
   ③  others.
